### PR TITLE
Bug fix + new function

### PR DESCRIPTION
**Pull request by SkepticMystic**
- [x] I agree to the contribution guidelines laid out [here](https://github.com/obsidian-community/obsidian-community-lib#contributing) in the readme.

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -320,6 +320,21 @@ export function linkedQ(
...
 }
 
+/**
+ * Check if the link `from` → `to` is resolved or not.
+ * @param  {App} app
+ * @param  {string} to
+ * @param  {string} from
+ * @returns boolean
+ */
+export function unresolvedQ(app: App, to: string, from: string): boolean {
```
Comment (phibr0) [inline src/utils.ts:331]: I think we should be using more self-explanatory variable/function names for this public library. I mean in most cases you can guess what it means from the tsdoc, but here I still don't know what `Q` ist for.

Comment (SkepticMystic) [reply src/utils.ts:331]: Fair, it's mostly a convention I use in my own code. `Q` is for Question. So `unresolvedQ` tells you if these links are unresolved. `linkedQ` tells you if these notes are linked.
I'm open to using a different naming convention though, what do you think?

Comment (phibr0) [reply src/utils.ts:331]: Not really sure, how about something like `notesLinked()`, or `linkResolved()`, of course that is the opposite of what this does

Comment (SkepticMystic) [reply src/utils.ts:331]: Obsidian uses `is` for boolean questions. So maybe `isLinked` or `isUnresolved` (which I easily turn into `isResolved`)

Comment (phibr0) [reply src/utils.ts:331]: Even better!

**Review by phibr0 — APPROVED** [review]

**Pull request merged**